test/EntityManagerFactoryTest.php

```diff
@@ -75,25 +79,35 @@ public function testConfigKeyTakenFromConfig() : void
                     ],
                 ],
             ],
-        ]);
-        $container->has('doctrine.connection.orm_foo')->willReturn(true);
-        $container->get('doctrine.connection.orm_foo')->willReturn($connection);
-        $container->has('doctrine.configuration.orm_bar')->willReturn(true);
-        $container->get('doctrine.configuration.orm_bar')->willReturn($configuration);
+        ];
+
+        $container = $this->getMockBuilder(ContainerInterface::class)->onlyMethods(['has', 'get'])->getMock();
```

Comment:
`$this->createMock()` suffices

Reply:
Addressed with bbe09841c12999e74a99b7af5d99804871fc90f2